common/test/acceptance/pages/lms/tab_nav.py

```diff
@@ -104,3 +104,8 @@ def _is_on_tab_promise(self, tab_name):
             lambda: self._is_on_tab(tab_name),
             "{0} is the current tab".format(tab_name)
         )
+
+    def has_new_post_button_visible_on_tab(self):
+        """Check if new post button present and visible on course tab page"""
```

Comment:
and here too: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings